server/data/statsClient.js

```diff
@@ -34,13 +34,40 @@ module.exports = {
     return transactionalClient.query(query)
   },
 
+  getRecatFromTo(startDate, endDate, prisonId, transactionalClient) {
+    const query = {
+      text: `
+        with cat_table as (
+          select booking_id,
+                 sequence_no,
+                 coalesce (form_response -> 'supervisor' ->'review' ->>'supervisorOverriddenCategory',
+                           form_response -> 'recat' -> 'decision' ->>'category') as cat
+          from form)
```

Comment:
I went with the temp table solution in the end , i.e. 'cat_table ' but hopefully it is not actually all that complex. Its like a subselect in the from clause in oracle